scripts/docker/webui/elektrad-demo/Dockerfile

```diff
@@ -10,6 +11,11 @@ WORKDIR /home/elektra
 
 # mount as root user
 RUN kdb mount --with-recommends hosts user/hosts hosts
+RUN kdb mount /etc/networks system/networks hosts
+RUN kdb mount /etc/ssh/ssh_config system/ssh line
+RUN kdb mount /etc/ldap/ldap.conf system/ldap line
+RUN kdb mount /var/lib/dpkg/available system/dpkg/available dpkg
+RUN kdb mount /var/lib/dpkg/status system/dpkg/status dpkg
 
```

Comment:
you really want to join those statements together to speed up builds and reduce image size

Reply:
this is true for all your images